src/lib/controller/message_handler.ts

```diff
@@ -100,9 +100,15 @@ export class ControllerMessageHandler {
         return { success };
       }
       case ControllerCommand.beginExclusion: {
-        const success = await driver.controller.beginExclusion(
-          message.unprovision
-        );
+        const success =
+          message.unprovision !== undefined
+            ? await driver.controller.beginExclusion(message.unprovision)
+            : await driver.controller.beginExclusion(
+                message.strategy !== undefined
+                  ? { strategy: message.strategy }
+                  : undefined
+              );
+
         return { success };
       }
       case ControllerCommand.stopExclusion: {
```

Comment:
I'm missing the changes needed for this point in the migration guide:

> - The Controller.replaceFailedNode method overload accepting the node ID as the second parameter has been removed. Use the overload with the ReplaceNodeOptions object instead. NOTE: If you do not pass this object, the new node will be included without security.

Reply:
this was a deprecated signature that was removed. We switched to the new method a while ago, so there's no more that needs to be done here as far as I am aware.